docs/developer-release-signatures

```diff
@@ -0,0 +1,26 @@
+
+January 8, 2021
+
+Any of the following core Tahoe contributers may sign a release. Each
+release should be signed by at least two developers. They each
+independently produce a signature which are made available beside
+Tahoe releases after 1.15.0
+
+This statement is signed by the previous Tahoe release key. Any future
+such statements may be signed by it OR by any two developers (for
+example, to add or remove developers from the list).
+
+meejah
+0xC2602803128069A7
+9D5A 2BD5 688E CB88 9DEB CD3F C260 2803 1280 69A7
+https://meejah.ca/meejah.asc
+
+jean-paul calderone (exarkun)
+0xE27B085EDEAA4B1B
+96B9 C5DA B2EA 9EB6 7941 9DB7 E27B 085E DEAA 4B1B
+https://twistedmatrix.com/~exarkun/E27B085EDEAA4B1B.asc
+
+brian warner
+0xD43B4C9C73225AAF
+967E FE06 6998 7241 1A77 DF36 D43B 4C9C 7322 5AAF
+http://www.lothar.com/warner-gpg.html
```

Comment:
That's the key I use for email. For code-signing, let's use this one instead (which I use for signing Magic-Wormhole releases):

```
0x863333C265497810
5810 F125 7F8C F753 7753  895A 8633 33C2 6549 7810
http://www.lothar.com/warner-gpg.html
```

I'm still working on getting lothar.com switched over to HTTPS (I'm a horrible horrible webadmin). I'll let you know if/when I get that fixed so we can add the "`s`" to that URL.

I've sent this pubkey to Meejah in an offline conversation today, so he should confirm that this fingerprint matches what he received.

I'm ok with releases being signed by just one key, if other folks are. I'm probably also ok with key-change messages being signed by just one.. in some sense, our authority chain is more likely to be managed by packagers talking directly with one of us than by them counting signatures on other artifacts.

Reply:
HTTPS is working, feel free to make that `https://www.lothar.com/warner-gpg.html`